Moya.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Moya"
-  s.version      = "15.0.0-alpha.1"
```

Comment:
afaik this will be set once you run the rake release command

Reply:
Oooh could be. Would it be an issue to have manually changed it?

Reply:
not sure, it might (?) since if there is nothing to commit it might be stuck on some command in the script? not really sure 😅

Reply:
I'll see and fix if needed. :)